packages/commons/src/fhir/transformers/input.ts

```diff
@@ -208,7 +208,7 @@ interface Certificate {
   collector?: RelatedPerson
   hasShowedVerifiedDocument?: boolean
   payments?: Array<Payment | null>
-  data?: string
+  certificateTemplateId?: string
```

Comment:
```suggestion
  certificateTemplateId: string
```

Reply:
Is there any scenario when `collector` or `hasShowedVerifiedDocument` is not present on the certificate? Based on the pattern of the previous properties, I have made `certificateTemplateId` optional.

Reply:
That's a great question. If we could always trust that the optional operator is only applied when there truly is a chance of something being optional, then we could now infer that yes, indeed it can be that `collector` and `hasShowedVerifiedDocument` can be optional. 
Trusting the developer before us has thought this through, we can now do the same and say `certificateTemplateId` will always be there as a certificate cannot be printed without selecting a certificate. "Defensive programming" really is an anti-pattern and makes reading and understanding what the code does harder